R/labels.r

```diff
@@ -51,15 +63,17 @@ labs <- function(...) {
 xlab <- function(label) {
   labs(x = label)
 }
+
 #' @rdname labs
 #' @export
 ylab <- function(label) {
   labs(y = label)
 }
+
 #' @rdname labs
 #' @export
-ggtitle <- function(label) {
-  labs(title = label)
+ggtitle <- function(label, subtitle=NULL) {
```

Comment:
will fix (remind me never to poke CRAN about their barriers to entry ;-)

On Mon, Mar 14, 2016 at 6:32 PM, Hadley Wickham <notifications@github.com>
wrote:

> In R/labels.r
> <https://github.com/hadley/ggplot2/pull/1582#discussion_r56087824>:
>
> >  #' @rdname labs
> >  #' @export
> > -ggtitle <- function(label) {
> > -  labs(title = label)
> > +ggtitle <- function(label, subtitle=NULL) {
>
> Missing spaces around =
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hadley/ggplot2/pull/1582/files#r56087824>.
>